samples/test/features/social-login-mfa.feature

```diff
@@ -2,16 +2,12 @@ Feature: Direct Auth Social Login with MFA
   
   Background:
     Given a SPA, WEB APP or MOBILE Policy that defines Password as the only factor required for authentication
-    # And a configured IDP object for Facebook
-    # And an IDP routing rule defined to allow users in the Sample App to use the IDP
-      And the Application Sign on Policy is set to "Password + Another Factor"
+    And the Application Sign on Policy is set to "Password + Another Factor"
     And a user named "Mary"
-    # And Mary does not have an account in the org
 
-  Scenario: Mary logs in with a social IDP and gets an error message
-    Given Mary navigates to Basic Social Login View
-    When she clicks the "Login with Facebook" button
-    And logs in to Facebook
+  Scenario: Mary logs in with a Okta OIDC IDP and gets an error message
+    Given Mary navigates to the Login View
+    When she clicks the "Login with Okta OIDC IDP" button
+    And logs in to Okta OIDC IDP
     # And the remediation returns "MFA_REQUIRED"
```

Comment:
This test is failing on bacon, but passing locally. This is NOT a critical test, since we're testing a feature we DON'T support i.e. MFA with 3rd party IdP. So, there's no harm disabling it for now.